### DISCUSSION TITLE
Expecting value: line 1 column 1 (char 0)

### DISCUSSION THREAD

**Discussion by elmoypepin** [diStyApps/seait#26] [Q&A]
91 / 5,000
sorry for bothering but do you know why this comes out and it won't let me enter the automatic 1111
Installing requirements
Expecting value: line 1 column 1 (char 0)

Launching Web UI with arguments: --autolaunch --theme=dark --force-enable-xformers --xformers
\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

**Comment by diStyApps** [comment]
I think its related to auto1111, your config.json might be corrupted.
create a back and try to delete it.

**Reply by elmoypepin** [reply]
thank you and thanks for the application
